bigtable/tests/system.py

```diff
@@ -356,6 +356,32 @@ def _write_to_row(self, row1=None, row2=None, row3=None, row4=None):
         cell4 = Cell(CELL_VAL4, timestamp4)
         return cell1, cell2, cell3, cell4
 
+    def test_mutate_rows(self):
+        row_1 = self._table.row(ROW_KEY)
+        row_1.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, CELL_VAL1)
+        row_1.commit()
+        row_2 = self._table.row(ROW_KEY_ALT)
+        row_2.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, CELL_VAL2)
+        row_2.commit()
+        rows = [row_1, row_2]
+        self.rows_to_delete.extend(rows)
```

Comment:
Please add to `rows_to_delete` **before** calling `commit`, since `row_2.commit()` could fail before `row_1` was adding to `rows_to_delete`.

Also, you can just use `row1` and `row2` (i.e. no underscore needed)